static/music_ui.js

```
@@ -0,0 +1,389 @@
+        });
+        return aplayerLoadPromise;
+    };
+
+    // --- 5. 播放器挂载逻辑 (保留所有 DOM 安全赋值) ---
+    const executePlay = async (trackInfo, currentToken, shouldAutoPlay = true) => {
+        if (currentToken !== latestMusicRequestToken)
+            return;
+
+        if (getMusicPlayerInstance())
+            destroyMusicPlayer(true);
+        document.querySelectorAll('.music-player-bar.fading-out').forEach(el => el.remove());
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)